sea-orm-rocket/lib/src/database.rs

```diff
@@ -205,6 +208,16 @@ impl<'a, D: Database> Connection<'a, D> {
     }
 }
 
+impl<'r, D: Database> OpenApiFromRequest<'r> for Connection<'r, D> {
```

Comment:
As such, this block should be feature guarded.

Reply:
Added a "cfg" notation.